src/sui-link-item/index.jsx

```diff
@@ -11,6 +11,7 @@ export default class LinkItem extends Component {
       handleClick: PropTypes.func,
       icon: PropTypes.element,
       literal: PropTypes.string,
+      nofollow: PropTypes.bool,
```

Comment:
What about instead doing it a `nofollow` proptype having a `rel` proptype to define whatever `rel` you want, for example, 'prev'. I know there're not much options, but it will do more extensible our component.